### DISCUSSION TITLE
Show detailed beatmap information under beatmap tooltip in multiplayer lobbies

### DISCUSSION THREAD

**Discussion by smallketchup82** [ppy/osu#26015] [Ideas] [answered]
At the moment, on Lazer, multiplayer lobbies don't show any difficulty information about queued beatmaps anywhere. While you can right click them and press on details to view the difficulty information on the web, this doesn't update to account for applied mods (such as dt).

Because of this, it would be wise to add this information under the tooltip that already exists (at the moment, it only shows star level though) when hovering over the circle.
![image](https://github.com/ppy/osu/assets/69545310/505ef80b-a786-4867-88e5-da35d7eaa720)
This would achieve the same effect as hovering over a beatmap in stable multiplayer lobbies and being able to see the difficulty stats there.

The information that should be present is the AR, HP, CS, OD, BPM, length, and # of objects. Could also add the pass % but I'm not sure if the client is able to access that. I wouldn't mind implementing this myself. This discussion is more so to discuss this proposal and to discuss designs.

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/11306.

**Reply by smallketchup82** [reply]
Thanks!
